templates/vue/src/components/lightbox/H5PIframe.vue

```diff
@@ -150,21 +150,11 @@ export default {
         h5pVideo.setCaptionsTrack(settings.caption)
       }
     },
-    handlePlay(node) {
+    handlePlay() {
       this.$emit("show-play-screen", false)
-      const { id, mediaType } = node
-      thisTapestryTool.updateMediaIcon(id, mediaType, "pause")
-      thisTapestryTool.recordAnalyticsEvent("user", "play", "h5p-video", id, {
-        time: node.typeData.progress[0].value * node.mediaDuration,
-      })
```

Comment:
Why has this been removed?

Reply:
I removed all references to thisTapestryTool to avoid any errors, but I think I have yet to add this functionality back in. Thanks for pointing it out!